build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+import org.jetbrains.changelog.markdownToHTML
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+fun properties(key: String) = project.findProperty(key).toString()
+
+plugins {
+    // Kotlin support
+    kotlin("jvm") version "1.6.10"
+    // Gradle IntelliJ Plugin
+    id("org.jetbrains.intellij") version "1.3.1"
+    // Gradle Changelog Plugin
+    id("org.jetbrains.changelog") version "1.3.1"
+    // Gradle Qodana Plugin
+    id("org.jetbrains.qodana") version "0.1.13"
+    jacoco
+}
+
+group = properties("pluginGroup")
+version = properties("pluginVersion")
+
+// Configure project's dependencies
+repositories {
+    mavenCentral()
+}
+
+// Configure Gradle IntelliJ Plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
+intellij {
+    pluginName.set(properties("pluginName"))
```

Comment:
Previously this was `pluginName = project.name`(pydantic-pycharm-plugin), but this change makes it "Pydantic", I'm not sure what you would want here.

Reply:
Me too, I forget about the code😅
We should remove the code.